setup_heroku.md

```diff
@@ -8,4 +8,24 @@ to generate a script which will create all Heroku apps.
 Please review the script before running it and execute only the commands you need and understand.
 If you don't know what a command does: read the documentation and then execute it.
 
-If you think that the script is outdate please open a Pull Request.
+If you think that the script is outdated please open a Pull Request.
+
+### Transferring Ownership of the application
+
+After creating a Heroku app, you have to transfer the ownership to `admin@renuo.ch`. This step may already be required
+to execute certain commands (like setting a `renuoapp.ch` subdomain) on the Heroku CLI.
+
+Transfer the ownership as follows:
+
+1. Login to [Heroku](https://id.heroku.com/login) and go to your app
```

Comment:
Yes, seems so: https://devcenter.heroku.com/articles/transferring-apps

However, the script needs to be split into two, resulting in the following workflow:

1. The first script creates the app and requests transfer. 
1. The Heroku admin accepts the transfer
1. The second script runs all commands which require a verified account